src/library_pthread.js

```diff
@@ -212,7 +212,7 @@ var LibraryPThread = {
       // Call thread init functions (these are the emscripten_tls_init for each
       // module loaded.
       for (var i in PThread.tlsInitFunctions) {
-        PThread.tlsInitFunctions[i]();
+        if (Pthread.tlsInitFunctions.hasOwnProperty(i)) PThread.tlsInitFunctions[i]();
```

Comment:
Ooop, looks like there is still a typo.  Should be "PThread".

Reply:
Wow, well, amazing that you are this due dilligent with pullrequests. Sorry for the mess 🙈

Reply:
Are all tests supposed to be passing? I'm having trouble understanding the logs, seem to be some python stack traces mostly 🤔